react/src/pages/user/courses/EnrolledCoursesPage.jsx

```
@@ -0,0 +1,57 @@
+
+    const courses = data?.data || [];
+
+    return (
+        <div className={styles.container}>
+            <div className={styles.header}>
+                <h1>{t('enrolledCourses.myCourses')}</h1>
+                <div className={styles.tabs}>
+                    <button
+                        className={`${styles.tab} ${activeTab === 'IN_PROGRESS' ? styles.active : ''}`}
+                        onClick={() => setActiveTab('IN_PROGRESS')}
+                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)